packages/entity-server/src/routes/hierarchy/types.ts

```diff
@@ -17,10 +17,10 @@ export interface MultiEntityRequestParams {
   hierarchyName: string;
 }
 
-export type RequestBody = Record<string, unknown>;
+export type RequestBody = Record<string, never>;
```

Comment:
So this base type is now an empty request body?

Reply:
Yup! In some ways you could argue `unknown` is more appropriate, but I like the strictness that `never` add. Now if you try to attach the `attachMultipleEntities` middleware to a `SingleEntityRoute` you get a typescript error 👍